scripts/powerpin-export.py

```diff
@@ -1,88 +1,318 @@
+# Copyright (c) 2022 arch1t3cht
```

Comment:
I fixed the parsing in persp-mo, so this entire file can probably be removed now.